docs/tutorials/offline.md

```diff
@@ -331,11 +331,11 @@ the title of the content and the time we downloaded it, but the metadata can
 contain anything you want. The metadata is optional, so you can ignore it if
 you want.
 
-`storage.store` returns an IAbortableOperation that resolves to a
+`storage.store` returns an `IAbortableOperation` that resolves to a
 `shaka.externs.StoredContent` instance (a summary of the stored content).
 
-At this point, the content is now stored offline and it's ready to be played.
-Next we will add functionality to play offline content.
+At this point, the content is now stored offline and's ready to be played.
```

Comment:
"and's" is not a word.

Reply:
It's a contraction of __*and*__ and __*is*__. I'm not sure how widely it is used though. Will make it "*and is*".

Reply:
Let us clarify, then, that it's not in use in American English.  We can't make any claims about the way English may be spoken elsewhere, but the American 30-somethings on this team have never seen this particular contraction.

Sometimes even England surprises me, and some of England's former colonies retain old features of English that nobody else uses any more.  So who knows what to expect with this crazy language?  :-)

Reply:
Totally get your point. I have sort of this habit of writing __and's__ for __and is__ & __and has__, depends upon the tense. The main motive was to replace __and it's__ with __and is__, but my weird writing style made me go with the contraction.